protocol/src/main/java/io/camunda/zeebe/protocol/ZbColumnFamilies.java

```diff
@@ -15,155 +15,165 @@
 
-  EXPORTER,
+  EXPORTER(40),
 
-  AWAIT_WORKLOW_RESULT,
+  AWAIT_WORKLOW_RESULT(41),
 
-  JOB_BACKOFF,
+  JOB_BACKOFF(42),
 
   @Deprecated
-  DEPRECATED_DMN_DECISIONS,
+  DEPRECATED_DMN_DECISIONS(43),
   @Deprecated
-  DEPRECATED_DMN_DECISION_REQUIREMENTS,
+  DEPRECATED_DMN_DECISION_REQUIREMENTS(44),
   @Deprecated
-  DEPRECATED_DMN_LATEST_DECISION_BY_ID,
+  DEPRECATED_DMN_LATEST_DECISION_BY_ID(45),
   @Deprecated
-  DEPRECATED_DMN_LATEST_DECISION_REQUIREMENTS_BY_ID,
+  DEPRECATED_DMN_LATEST_DECISION_REQUIREMENTS_BY_ID(46),
   @Deprecated
-  DEPRECATED_DMN_DECISION_KEY_BY_DECISION_REQUIREMENTS_KEY,
+  DEPRECATED_DMN_DECISION_KEY_BY_DECISION_REQUIREMENTS_KEY(47),
   @Deprecated
-  DEPRECATED_DMN_DECISION_KEY_BY_DECISION_ID_AND_VERSION,
+  DEPRECATED_DMN_DECISION_KEY_BY_DECISION_ID_AND_VERSION(48),
   @Deprecated
-  DEPRECATED_DMN_DECISION_REQUIREMENTS_KEY_BY_DECISION_REQUIREMENT_ID_AND_VERSION,
+  DEPRECATED_DMN_DECISION_REQUIREMENTS_KEY_BY_DECISION_REQUIREMENT_ID_AND_VERSION(49),
 
   // signal subscription
-  SIGNAL_SUBSCRIPTION_BY_NAME_AND_KEY,
-  SIGNAL_SUBSCRIPTION_BY_KEY_AND_NAME,
+  SIGNAL_SUBSCRIPTION_BY_NAME_AND_KEY(50),
+  SIGNAL_SUBSCRIPTION_BY_KEY_AND_NAME(51),
 
   // distribution
-  PENDING_DISTRIBUTION,
-  COMMAND_DISTRIBUTION_RECORD,
-  MESSAGE_STATS,
+  PENDING_DISTRIBUTION(52),
+  COMMAND_DISTRIBUTION_RECORD(53),
 
-  PROCESS_INSTANCE_KEY_BY_DEFINITION_KEY,
+  MESSAGE_STATS(54),
+
+  PROCESS_INSTANCE_KEY_BY_DEFINITION_KEY(55),
 
   // This was introduces in 8.3 and backported to earlier versions. As this turns out to not be safe
   // to do we have removed the usage of this CF. We must keep it to remain backwards compatible.
   @Deprecated
-  MIGRATIONS_STATE,
+  MIGRATIONS_STATE(56),
+  PROCESS_VERSION(57),
+  PROCESS_CACHE(58),
+  PROCESS_CACHE_BY_ID_AND_VERSION(59),
+
+  PROCESS_CACHE_DIGEST_BY_ID(60),
+  DMN_DECISIONS(61),
+  DMN_DECISION_REQUIREMENTS(62),
+  DMN_LATEST_DECISION_BY_ID(63),
+  DMN_LATEST_DECISION_REQUIREMENTS_BY_ID(64),
+  DMN_DECISION_KEY_BY_DECISION_REQUIREMENTS_KEY(65),
+  DMN_DECISION_KEY_BY_DECISION_ID_AND_VERSION(66),
+
+  DMN_DECISION_REQUIREMENTS_KEY_BY_DECISION_REQUIREMENT_ID_AND_VERSION(67),
+  FORMS(68),
+  FORM_VERSION(69),
+
+  FORM_BY_ID_AND_VERSION(70),
+  MESSAGES(71),
+  MESSAGE_START_EVENT_SUBSCRIPTION_BY_NAME_AND_KEY(72),
+  MESSAGE_START_EVENT_SUBSCRIPTION_BY_KEY_AND_NAME(73),
+  MESSAGE_SUBSCRIPTION_BY_NAME_AND_CORRELATION_KEY(74),
 
-  PROCESS_VERSION,
-  PROCESS_CACHE,
-  PROCESS_CACHE_BY_ID_AND_VERSION,
-  PROCESS_CACHE_DIGEST_BY_ID,
+  PROCESS_SUBSCRIPTION_BY_KEY(75),
 
-  DMN_DECISIONS,
-  DMN_DECISION_REQUIREMENTS,
-  DMN_LATEST_DECISION_BY_ID,
-  DMN_LATEST_DECISION_REQUIREMENTS_BY_ID,
-  DMN_DECISION_KEY_BY_DECISION_REQUIREMENTS_KEY,
-  DMN_DECISION_KEY_BY_DECISION_ID_AND_VERSION,
-  DMN_DECISION_REQUIREMENTS_KEY_BY_DECISION_REQUIREMENT_ID_AND_VERSION,
+  JOB_ACTIVATABLE(76);
 
-  FORMS,
-  FORM_VERSION,
-  FORM_BY_ID_AND_VERSION,
+  private final int value;
```

Comment:
🔧 Same here I would expect to simply copy over the complete enum, this doesn't do any harm and is much simpler to backport in the future.

Reply:
✍️ For completeness, we discussed this here: https://github.com/camunda/zeebe/pull/16129#discussion_r1472809402